android/src/main/java/com/iterable/reactnative/RNIterableAPIModule.java

```diff
@@ -113,9 +113,9 @@ public void setEmail(@Nullable String email) {
     }
 
     @ReactMethod
-    public void updateEmail(String email) {
+    public void updateEmail(String email, @Nullable String authToken) {
         IterableLogger.d(TAG, "updateEmail: " + email);
-        IterableApi.getInstance().updateEmail(email);
+        IterableApi.getInstance().updateEmail(email, authToken);
```

Comment:
Might be mistaken but I am not seeing updateEmail method with authToken in our Android SDK. Its just setEmail and setUserID.. strange coz i felt I've seen it recently 😅

Reply:
Yeah, I just put it in ahead of time.